### PR TITLE
Add a couple of older Juniper devices

### PR DESCRIPTION
**Pull request by mihailim**
Add device types:

* EX2200-24T-4G
* NetScreen SSG-140

### CODE REVIEW

File: device-types/Juniper/EX2200-24T-4G.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+manufacturer: Juniper
+model: EX2200-24T-4G
+slug: juniper-ex2200-24t-4g
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/Juniper/EX2200-24T-4G.yaml:6]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/Juniper/NetScreen-SSG-140.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Juniper
+model: NetScreen SSG-140
+slug: juniper-netscreen-ssg-140
+u_height: 1.0
+is_full_depth: false
+airflow: front-to-rear
```
Comment (danner26) [inline device-types/Juniper/NetScreen-SSG-140.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Juniper/EX2200-24T-4G.yaml (hunk 1 above)
Comment (mihailim) [reply device-types/Juniper/EX2200-24T-4G.yaml:8]: Thank you, done!

File: device-types/Juniper/NetScreen-SSG-140.yaml (hunk 2 above)
Comment (mihailim) [reply device-types/Juniper/NetScreen-SSG-140.yaml]: Thank you, done! Also fixed the previously incorrect airflow spec.

**Pull request merged**